src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -426,6 +427,17 @@ def _upload_resources(
                 # resource creation succeeded: update the iri_resolver and remove the resource from the list
                 iri, label = res
                 _tidy_up_resource_creation(iri, label, iri_resolver, resource, current_res, total_res)  # type: ignore[arg-type]
+        except PermanentConnectionError:
```

Comment:
The error handling is a bit stupid, but because of the broad excepts later for both I need to handle this specific error twice...